test/standalone_test.dart

```diff
@@ -374,46 +374,85 @@ void main() {
+            await (await grind(["pkg-standalone-linux-riscv64"])).shouldExit(0);
+            await archive("linux", "riscv64", musl: musl).validate();
+          });
+        });
+      }
+    });
+
+    group("Windows", () {
+      test("32-bit x86", () async {
+        await (await grind(["pkg-standalone-windows-ia32"])).shouldExit(0);
+        await archive("windows", "ia32").validate();
       });
 
       test("64-bit x86", () async {
-        await (await grind(["pkg-standalone-linux-x64"])).shouldExit(0);
-        await archive("linux", "x64").validate();
+        await (await grind(["pkg-standalone-windows-x64"])).shouldExit(0);
+        await archive("windows", "x64").validate();
       });
 
       test("64-bit ARM", () async {
-        await (await grind(["pkg-standalone-linux-arm64"])).shouldExit(0);
-        await archive("linux", "arm64").validate();
+        await (await grind(["pkg-standalone-windows-arm64"])).shouldExit(0);
+        await archive("windows", "arm64").validate();
       });
     });
 
-    group("Windows", () {
-      test("32-bit", () async {
-        await (await grind(["pkg-standalone-windows-ia32"])).shouldExit(0);
-        await archive("windows", "ia32").validate();
+    group("Android", () {
+      test("32-bit x86", () async {
+        await (await grind(["pkg-standalone-android-ia32"])).shouldExit(0);
+        await archive("android", "ia32").validate();
       });
 
-      test("64-bit", () async {
-        await (await grind(["pkg-standalone-windows-x64"])).shouldExit(0);
-        await archive("windows", "x64").validate();
+      test("64-bit x86", () async {
+        await (await grind(["pkg-standalone-android-x64"])).shouldExit(0);
+        await archive("android", "x64").validate();
+      });
+
+      test("32-bit ARM", () async {
+        await (await grind(["pkg-standalone-android-arm"])).shouldExit(0);
+        await archive("android", "arm").validate();
+      });
+
+      test("64-bit ARM", () async {
+        await (await grind(["pkg-standalone-android-arm64"])).shouldExit(0);
+        await archive("android", "arm64").validate();
       });
     });
 
-    test("all platforms", () async {
-      await (await grind(["pkg-standalone-all"])).shouldExit(0);
-
-      await Future.wait([
-        archive("macos", "x64").validate(),
-        archive("macos", "arm64").validate(),
-        archive("linux", "ia32").validate(),
-        archive("linux", "x64").validate(),
-        archive("linux", "arm64").validate(),
-        archive("windows", "ia32").validate(),
-        archive("windows", "x64").validate()
-      ]);
+    group("iOS", () {
+      test("64-bit ARM", () async {
+        await (await grind(["pkg-standalone-ios-arm64"])).shouldExit(0);
+        await archive("ios", "arm64").validate();
+      });
+
+      test("64-bit x86", () async {
+        await (await grind(["pkg-standalone-ios-x64"])).shouldExit(0);
+        await archive("ios", "x64").validate();
+      });
```

Comment:
What are these builds useful for, if we can't build native binaries cross-platform and there's no iOS Dart SDK? Also, what x86 device runs iOS?

Reply:
Good point. I'll just cut the iOS releases until that changes.